ts/private/ts_lib.bzl

```diff
@@ -22,7 +22,16 @@ STD_ATTRS = {
         doc = "https://www.typescriptlang.org/docs/handbook/compiler-options.html",
     ),
     "data": attr.label_list(
-        doc = "Runtime dependencies to include in binaries/tests that depend on this TS code",
+        doc = """Runtime dependencies to include in binaries/tests that depend on this library.
+
+        If this list contains linked npm packages, npm package store targets or other targets that provide 'JsInfo',
```

Comment:
why so much single quotes instead of backticks? won't render properly as code

Reply:
VsCode :(

I should figure out the regex to fix them all

Reply:
Cleaned up with regex.